common/static/karma_common.conf.js

```diff
@@ -38,7 +38,7 @@ var libraryFiles = [
     {pattern: 'js/vendor/jquery.truncate.js', included: true},
     {pattern: 'js/vendor/mustache.js', included: true},
     {pattern: 'common/js/vendor/underscore.js', included: true},
-    {pattern: 'js/vendor/underscore.string.min.js', included: true},
```

Comment:
I remember seeing changes to some underscore paths during one of the last rebases of the feature branch. It is possible this path wasn't changed correctly.

Only `karma_common.conf.js` has `included:true` so the other suites must be either requiring the correct file or do not need this library.